lib/Conversion/TTIRToTTNN/TTIRToTTNN.cpp

```diff
@@ -822,7 +822,23 @@ class SubtractOpConversionPattern
           srcOp, adaptor.getInputs().front(), negOp.getResults().front(),
           adaptor.getOutputs().front());
     }
+    return success();
+  }
+};
 
+template <typename TTIROpTy, typename TTNNOpTy,
+          typename OpAdaptor = typename TTIROpTy::Adaptor>
+class SelectOpConversionPattern : public OpConversionPattern<TTIROpTy> {
```

Comment:
Pattern is called `SelectOpConversionPattern` but it converts `ttir::WhereOp` to `ttnn::WhereOp`.